.github/workflows/update_codeowners.yml

```
@@ -0,0 +1,39 @@
+name: update codeowners
+on:
+  schedule:
+    - cron: 0 * * * *
+  workflow_dispatch:
+jobs:
+  update:
+    runs-on: ubuntu-20.04
+    if: github.repository == 'web-platform-tests/wpt'
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+    - name: Checkout wpt
+      uses: actions/checkout@v4
+      with:
+        path: wpt
+    - name: Checkout wpt-metadata
+      uses: actions/checkout@v4
+      with:
+        path: wpt-metadata
+        repository: web-platform-tests/wpt-metadata
+    - name: Update codeowners
+      run: |
+        cd wpt
+        python wpt update-codeowners ../wpt-metadata
```

Wouldn't this work and be more like our docs?

```suggestion
        ./wpt update-codeowners ../wpt-metadata
```